app/data/models/dog.js

```diff
@@ -7,6 +7,10 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: false,
       primaryKey: true
     },
+    orig_id: {
```

Comment:
I'm assuming this is for the index number? If it is I think it might be better to rename this column to something like `orig_index_number`.

It might also be a good idea to allow this to be null, as new dogs onto the index might not have an original index number

I think this column should also be on the `registration` model as the `dog` model just represents the dog itself, rather than it's entry on the index.

Reply:
Good point about it needing to allow null, and I agree with the name change. It's the dog index number (not the person index or access db row index), so if we want to keep the info it seems it should be in the dog model. I'll amend the null and name.